orion/sites/admin/src/index.js

```diff
@@ -63,6 +69,9 @@ class App extends React.Component {
           <Route path="/locations/:locId/edit" component={LocationEditMatch}/>
           <Route path="/locations/add" component={LocationEdit}/>
           <Route path="/locations" component={Location}/>
+          <Route path="/semesters/:locationId/edit" component={SemesterEditMatch}/>
```

Comment:
You have `:locationId` here. Probably `semesterId`?

Reply:
Oops